Automattic/wp-calypso
# Components: Update WpcomColophon to use an independent SVG

#### Changes proposed in this Pull Request

* Update `<WpcomColophon />` to use a dedicated SVG, as suggested here: https://github.com/Automattic/wp-calypso/pull/30721#issuecomment-465315219.

#### Preview
Before (in both **Classic Bright** and **Laser Black** color schemes):
![](https://cldup.com/HFJR5eM0MP.png)
![](https://cldup.com/Tdg0LT4NqQ.png)

After (in both **Classic Bright** and **Laser Black** color schemes):
![](https://cldup.com/mq350VJsGt.png)
![](https://cldup.com/JqoGDCjuZk.png)

#### Testing instructions

* Go to `<WpcomColophon />`'s devdocs at calypso.live from [this link](https://calypso.live/devdocs/design/wpcom-colophon?branch=update/wpcom-colophon-use-svg).
* Verify the component looks well and as shown on the screenshots above.
* Play with the color schemes on that page, verify it looks well.

#### Questions:

* This currently uses a single SVG and the global text color `var( --color-text )` for defining the color. Is that alright? Is there a better color variable we could use here? I'm asking because [Initial suggestion](https://github.com/Automattic/wp-calypso/pull/30721#issuecomment-465315219) provided 2 separate SVGs for light and dark color schemes, and I'd like to avoid using 2 separate SVGs for this.
